src/pages/home/report/ReportActionCompose/ReportActionCompose.tsx

```diff
@@ -192,6 +217,16 @@ function ReportActionCompose({
     const includesConcierge = useMemo(() => chatIncludesConcierge({participants: report?.participants}), [report?.participants]);
     const userBlockedFromConcierge = useMemo(() => isBlockedFromConciergeUserAction(blockedFromConcierge), [blockedFromConcierge]);
     const isBlockedFromConcierge = useMemo(() => includesConcierge && userBlockedFromConcierge, [includesConcierge, userBlockedFromConcierge]);
+    const shouldDisplayDualDropZone = useMemo(
+        () => !isChatRoom(report) && !isUserCreatedPolicyRoom(report) && !isAnnounceRoom(report) && !isAdminRoom(report) && !isConciergeChatReport(report) && !isInvoiceReport(report),
```

Comment:
```suggestion
        () => !isChatRoom(report) && !isUserCreatedPolicyRoom(report) && !isAnnounceRoom(report) && !isAdminRoom(report) && !isConciergeChatReport(report) && !isInvoiceReport(report) && !isGroupChat(report),
```

This or i wonder if we can trigger split expense flow.
thoughts? @lakchote @JmillsExpensify